sci-calculators/qalculate-gtk/qalculate-gtk-5.3.0.ebuild

```diff
@@ -37,3 +37,7 @@ src_prepare() {
 
 	default
 }
+
+pkg_postinst() {
```

Comment:
Since we now explicitly define `pkg_postinst` we must ensure that `xdg_pkg_postinst` is still called here. Previously it was called implicitly since the phase was not defined by the ebuild.

Reply:
Added with the latest commit!